app/models/video.py

```diff
@@ -1,4 +1,20 @@
 from app import db
 
 class Video(db.Model):
-    id = db.Column(db.Integer, primary_key=True)
\ No newline at end of file
+
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    title = db.Column(db.String)
+    release_date = db.Column(db.DateTime())
+    total_inventory = db.Column(db.Integer)
+
+    def to_dict(self):
+        return {
+            "id": self.id,
+            "title" : self.title,
+            "release_date" : self.release_date,
+            "total_inventory" : self.total_inventory
+        }       
+
+    @classmethod
+    def from_dict(cls, values):
+        return cls(**values)
```

Comment:
The same comment about _required information_ for `Customer` can also be applied to `Video` and `Rental`.